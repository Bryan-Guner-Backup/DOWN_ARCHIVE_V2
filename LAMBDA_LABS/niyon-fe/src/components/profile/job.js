export const job = [
  {
    value: 1,
    label: 'Unknown'
  },
  {
    value: 2,
    label: 'Full Stack Web Developer'
  },
  {
    value: 3,
    label: 'Data Scientist'
  },
  {
    value: 4,
    label: 'UI Designer'
  },
  {
    value: 5,
    label: 'iOS Developer'
  },
  {
    value: 6,
    label: 'Android Developer'
  },
  {
    value: 7,
    label: 'Network Programmer'
  },
  {
    value: 8,
    label: 'AWS Solutions Architect'
  },
  {
    value: 9,
    label: 'Front-End Developer'
  },
  {
    value: 10,
    label: 'Back-End Developer'
  },
  {
    value: 11,
    label: 'Software Engineer'
  },
  {
    value: 12,
    label: 'Junior Developer'
  },
  {
    value: 13,
    label: 'Senior Developer'
  },
  {
    value: 14,
    label: 'Software Architect'
  },
  {
    value: 15,
    label: 'Dev Ops Specialist'
  },
  {
    value: 16,
    label: 'Java Developer'
  },
  {
    value: 17,
    label: 'SQL Developer'
  },
  {
    value: 18,
    label: '.NET Developer'
  },
  {
    value: 19,
    label: 'Python Developer'
  },
  {
    value: 20,
    label: 'Web Analytics Developer'
  },
  {
    value: 21,
    label: 'UX Designer'
  },
  {
    value: 22,
    label: 'Ruby Developer'
  },
  {
    value: 23,
    label: 'Systems Engineer'
  },
  {
    value: 24,
    label: 'Project Manager'
  },
  {
    value: 25,
    label: 'Director of Technology'
  },
  {
    value: 26,
    label: 'Webmaster'
  },
  {
    value: 27,
    label: 'Student'
  },
  {
    value: 28,
    label: 'Freelancer'
  },
  {
    value: 29,
    label: 'Unknown'
  },
  {
    value: 30,
    label: 'Full Stack Web Developer'
  },
  {
    value: 31,
    label: 'Data Scientist'
  },
  {
    value: 32,
    label: 'UI Designer'
  },
  {
    value: 33,
    label: 'iOS Developer'
  },
  {
    value: 34,
    label: 'Android Developer'
  },
  {
    value: 35,
    label: 'Network Programmer'
  },
  {
    value: 36,
    label: 'AWS Solutions Architect'
  },
  {
    value: 37,
    label: 'Front-End Developer'
  },
  {
    value: 38,
    label: 'Back-End Developer'
  },
  {
    value: 39,
    label: 'Software Engineer'
  },
  {
    value: 40,
    label: 'Junior Developer'
  },
  {
    value: 41,
    label: 'Senior Developer'
  },
  {
    value: 42,
    label: 'Software Architect'
  },
  {
    value: 43,
    label: 'Dev Ops Specialist'
  },
  {
    value: 44,
    label: 'Java Developer'
  },
  {
    value: 45,
    label: 'SQL Developer'
  },
  {
    value: 46,
    label: '.NET Developer'
  },
  {
    value: 47,
    label: 'Python Developer'
  },
  {
    value: 48,
    label: 'Web Analytics Developer'
  },
  {
    value: 49,
    label: 'UX Designer'
  },
  {
    value: 50,
    label: 'Ruby Developer'
  },
  {
    value: 51,
    label: 'Systems Engineer'
  },
  {
    value: 52,
    label: 'Project Manager'
  },
  {
    value: 53,
    label: 'Director of Technology'
  },
  {
    value: 54,
    label: 'Webmaster'
  },
  {
    value: 55,
    label: 'Student'
  },
  {
    value: 56,
    label: 'Freelancer'
  },
  {
    value: 57,
    label: 'Unknown'
  },
  {
    value: 58,
    label: 'Full Stack Web Developer'
  },
  {
    value: 59,
    label: 'Data Scientist'
  },
  {
    value: 60,
    label: 'UI Designer'
  },
  {
    value: 61,
    label: 'iOS Developer'
  },
  {
    value: 62,
    label: 'Android Developer'
  },
  {
    value: 63,
    label: 'Network Programmer'
  },
  {
    value: 64,
    label: 'AWS Solutions Architect'
  },
  {
    value: 65,
    label: 'Front-End Developer'
  },
  {
    value: 66,
    label: 'Back-End Developer'
  },
  {
    value: 67,
    label: 'Software Engineer'
  },
  {
    value: 68,
    label: 'Junior Developer'
  },
  {
    value: 69,
    label: 'Senior Developer'
  },
  {
    value: 70,
    label: 'Software Architect'
  },
  {
    value: 71,
    label: 'Dev Ops Specialist'
  },
  {
    value: 72,
    label: 'Java Developer'
  },
  {
    value: 73,
    label: 'SQL Developer'
  },
  {
    value: 74,
    label: '.NET Developer'
  },
  {
    value: 75,
    label: 'Python Developer'
  },
  {
    value: 76,
    label: 'Web Analytics Developer'
  },
  {
    value: 77,
    label: 'UX Designer'
  },
  {
    value: 78,
    label: 'Ruby Developer'
  },
  {
    value: 79,
    label: 'Systems Engineer'
  },
  {
    value: 80,
    label: 'Project Manager'
  },
  {
    value: 81,
    label: 'Director of Technology'
  },
  {
    value: 82,
    label: 'Webmaster'
  },
  {
    value: 83,
    label: 'Student'
  },
  {
    value: 84,
    label: 'Freelancer'
  }
]
