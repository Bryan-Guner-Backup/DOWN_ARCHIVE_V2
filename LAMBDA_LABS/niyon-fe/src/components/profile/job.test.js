import React from 'react'
import toJSON from 'enzyme-to-json'
import { job } from './job'

describe('Testing jobs', () => {
  it('should render jobs array correctly', () => {
    expect(job).toEqual(expect.arrayContaining(job))
  })

  it('should pass a snapshot test correctly', () => {
    expect(toJSON(job)).toMatchSnapshot()
  })
})
