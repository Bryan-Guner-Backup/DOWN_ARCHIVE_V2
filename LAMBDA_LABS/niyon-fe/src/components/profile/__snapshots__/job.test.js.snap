// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing jobs should pass a snapshot test correctly 1`] = `undefined`;
